#uint64 timestamp
float32[3] pos
float32[3] vel
float32[3] acc
float32[3] jerk
float32[3] heading
